src/cls/_ZPM/CLI.cls

```diff
@@ -108,14 +108,14 @@ ClassMethod %HelpForCommand(pCommandName As %String, ByRef pCommandStruct, pDeta
 	If pMarkdownFormat {
 		Write "h2. "
 	}
-	Write $$$EscapeOrFormat(pCommandName, ##class(%ZPM.PackageManager).TerminalPromptColor())
+	Write $$$EscapeOrFormat(pCommandName, ..TerminalPromptColor())
```

Comment:
%ZPM.Main is a subclass of this class so this sort of dependency isn't great. The TerminalPromptColor() method should be moved into this class and referenced using .. instead.

Reply:
Fixed